api.yaml

```diff
@@ -148,7 +148,7 @@ paths:
                 $ref: '#/components/schemas/IdResponse'
   /project:
     get:
-      operationId: project-list
+      operationId: projects
```

Comment:
disagree, it's an operation, not an entity:
- create-project
- get-project

why not get-projects / list-projects?